package/contents/ui/main.qml

```diff
@@ -28,11 +28,16 @@ PlasmoidItem {
     property bool widgetHovered: widgetHoverHandler.hovered
     property bool vertical: plasmoid.formFactor === PlasmaCore.Types.Vertical
     property bool leftEdgeLocation: plasmoid.location === PlasmaCore.Types.LeftEdge
+    property bool hideWidget: !tasksModel.hasActiveWindow && plasmoid.configuration.widgetElementsDisabledMode === WidgetElement.DisabledMode.Hide
+    property bool inEditMode: Plasmoid.containment.corona?.editMode ?? false
 
     signal invokeKWinShortcut(string shortcut)
     signal widgetElementsLayoutUpdated
 
     Plasmoid.constraintHints: Plasmoid.CanFillArea
+    Plasmoid.status: hideWidget
```

Comment:
It seems the `status` property is just tells to container that the plasmoid is hidden.
Let's remove the `emptyWidgetPlaceholder` to hide it.
This line does not require changes.

Reply:
Done